.changeset/tough-parrots-occur.md

```diff
@@ -0,0 +1,7 @@
+---
+"@fuel-ts/account": patch
+---
+
+- Implemented transaction signing through an Account
+- Add signers to a transaction request through `addAccountWitnesses`
+- Renamed `createWitness` -> `addEmptyWitness` and implemented `addWitness`
```

Comment:
#1872 got merged and the changeset description should just be the title of the PR, "feat!: improve signing DX".

Reply:
Okay I'll make the change, but this still feels a bit off to me

Reply:
https://github.com/FuelLabs/fuels-ts/pull/1815/commits/4d9baea755c81d101e9ce1b1659188729f51fb46